bot/index.ts

```diff
@@ -45,13 +66,193 @@ async function say_hello(bot: KaiaBotClient, event: MessageEvent) {
   }
 }
 
-async function fugafuga(bot: KaiaBotClient, event: MessageEvent) {
+async function receipt(bot: KaiaBotClient, event: MessageEvent) {
   try {
     const to = event.source.userId || "";
-    const messages: Array<TextMessage> = [
+    const messages: Array<messagingApi.FlexMessage> = [
       {
-        type: "text",
-        text: "fugafuga",
+        type: "flex",
+        altText: "recept",
+        contents: {
+          type: "bubble",
+          body: {
+            type: "box",
+            layout: "vertical",
+            contents: [
+              {
+                type: "text",
+                text: "RECEIPT",
+                weight: "bold",
+                color: "#1DB446",
+                size: "sm"
+              },
+              {
+                type: "text",
+                text: "Carol's Birthday Party",
```

Comment:
```suggestion
                text: "Kaia's Birthday Party",
```
How about Kaia's birthday party? Kaia mainnet launches today!

Reply:
That's good (: